geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommand.java

```diff
@@ -300,13 +300,16 @@ public boolean updateConfigForGroup(String group, CacheConfig cacheConfig, Objec
     if (regionConfig == null) {
       return false;
     }
-
     RegionAttributesType attributes = getRegionAttribute(regionConfig);
-    addMappingToRegion(regionMapping, regionConfig);
-    if (!synchronous) {
-      createAsyncQueue(cacheConfig, attributes, queueName);
+    if (MappingCommandUtils.isAccessor(attributes)) {
```

Comment:
Have you confirmed that "isAccessor" works on a cluster config region created using a RegionShortcut?